src/simulators/matrix_product_state/matrix_product_state_internal.cpp

```diff
@@ -199,17 +203,14 @@ std::vector<uint_t> calc_new_indices(const reg_t &indices) {
 }
 
 cmatrix_t mul_matrix_by_lambda(const cmatrix_t &mat,
-			       const rvector_t &lambda)
+			       const rvector_t &lambda,
+			       uint_t omp_threads)
```

Comment:
I think we are not using this variable anywhere, right?
Furthermore, shouldn't be a parameter of this function. Using concurrency in a multiplication operation should be an internal implementation detail and not exposed to the caller (through a parameter). The function itself, in it's implementation should have all the needed data to decide whether it can run concurrently.

Reply:
you are correct. done.